### DISCUSSION TITLE
Add models from another GDrive?

### DISCUSSION THREAD

**Discussion by DivanoDova** [TheLastBen/fast-stable-diffusion#1535] [Q&A] [answered]
Is there a way to load models folder from another Google Drive? I can no longer use my models because they take almost all the available space on Google Drive. I tried to use the shared link but this seems to not works

**Comment by TheLastBen** [comment, marked as answer]
the shared link should work, make sure the file is set to public

**Reply by DivanoDova** [reply]
I tried to put the shared link without success.
But i managed to make this works by sharing the folder to my alt account and then i put a shortcut of that foldr to my main Google Drive
Huge thank you for this repo :)

**Reply by umishima** [reply]
> I tried to put the shared link without success.
> But i managed to make this works by sharing the folder to my alt account and then i put a shortcut of that foldr to my main Google Drive
> Huge thank you for this repo :)

Can you please share details how to do so? having same problem. Thanks in advance!

**Reply by ghost** [reply]
this is how i got it to work.

1. have 2 google drive accounts. 1 (main) it will have your Auto1111 on it, 2 (models) this will be the one with your models.
2. on the 2nd drive create a folder named "models" or whatever you like. ( I prefer to copy and paste the original folders into here. Loras, stable, etc.)
3. add your models.
4. share that the "models" folder to the main drive and set it to public.
5. open the folder and click on each model you would like to add.
6. click the three dots at the top and click "add shortcut to drive".
7. locate the appropriate area and add the shortcut into there.

should should be good to go. hope this helps.

**Reply by OMeher** [reply]
> this is how i got it to work.
> 
> 1. have 2 google drive accounts. 1 (main) it will have your Auto1111 on it, 2 (models) this will be the one with your models.
> 2. on the 2nd drive create a folder named "models" or whatever you like. ( I prefer to copy and paste the original folders into here. Loras, stable, etc.)
> 3. add your models.
> 4. share that the "models" folder to the main drive and set it to public.
> 5. open the folder and click on each model you would like to add.
> 6. click the three dots at the top and click "add shortcut to drive".
> 7. locate the appropriate area and add the shortcut into there.
> 
> should should be good to go. hope this helps.

Thank you so much, with this I can load other models and use them simultaneously.
